app/policies/log_policy.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class LogPolicy < TeamDependantPolicy
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    @user.is_a?(User::Human) && team.teammember?(@user.id)
```

Comment:
the method 'team_member?' provided by TeamDepdendantPolicy does exactly this.
so just use it :wink: